christfellowshipchurch/web-app-v2
# Christmas Banner

### About
This PR implements the Christmas Banner that links to the Christmas at Christ Fellowship page.

_Note: If the location single changes are merged this should look good, but we might need to update some things._ 

### Test Instructions
1. Ensure that the banner on the [testing link](https://web-app-v2-git-2023-christmas-banner-christ-fellowship-church.vercel.app/locations/palm-beach-gardens) looks like the Figma mockup.
2. Ensure that the link in the banner takes the user to the correct URL.

### Screenshots
<img width="1189" alt="Screenshot 2023-12-08 at 4 05 40 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/6ee9b2fa-7476-4889-8905-b39561999416">


### Closes Tickets
[CFDP-2830](https://christfellowshipchurch.atlassian.net/browse/CFDP-2830)

[CFDP-2830]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2830?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
